README.md

```diff
@@ -76,11 +79,26 @@ You can adjust defaults by passing arguments to `pg-migrate`:
 
 See all by running `pg-migrate --help`.
 
-Most of configuration options can be also specified in `node-config` configuration file.
+Most of configuration options can be also specified in [config](https://www.npmjs.com/package/config) configuration file.
```

Comment:
config configuration file?
---> I would omit the middle "configuration" word here.